web/docs/deploying.md

```diff
@@ -31,7 +31,8 @@ Below we will explain the required env vars and also provide detailed instructio
 
 Server uses following environment variables, so you need to ensure they are set on your hosting provider:
 - `PORT` -> number of port at which it will listen for requests (e.g. `3001`).
-- `DATABASE_URL` -> url to the Postgres database that it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`)
+- `DATABASE_URL` -> URL to the Postgres database that it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).
```

Comment:
I know it's an old line, but I suggest changing it to:

```suggestion
- `DATABASE_URL` -> The URL of the Postgres database it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).
```
We could also capitalize the rest of them. If I correctly remember, that's the style we use in the rest of our docs.

Reply:
Sounds good I can review what is there for consistency, thanks.